bug_report_page.php

```diff
@@ -281,7 +281,7 @@
 				echo $t_allow_no_category ? '' : ' required';
 			?>>
 				<?php
-					print_category_option_list( $f_category_id );
+					print_category_option_list( $f_category_id, null, true);
```

Comment:
Missing space before `)` and I would add hints of parameters that are not clear:

```
print_category_option_list( $f_category_id, /* project_id */ null, /* active_only */ true );
```